starter-code/app.js

```diff
@@ -1,5 +1,31 @@
+'use strict';
+
 const express = require('express');
 const app = express();
-const Chuck  = require('chucknorris-io');
+const Chuck = require('chucknorris-io');
 const client = new Chuck();
+const expressLayouts = require('express-ejs-layouts');
+
+// configure app
+app.set('views', __dirname + '/views');
+app.set('view engine', 'ejs');
+
+app.use(expressLayouts);
+
```

Comment:
Here, in fact, you want to display the same thing that in /random (not only the index but with a joke), for doing that:
```
res.redirect('/random');
```